library/std/src/net/addr.rs

```diff
@@ -747,7 +747,7 @@ impl hash::Hash for SocketAddrV6 {
 ///
 ///  * `(`[`&str`]`, `[`u16`]`)`: the string should be either a string representation
 ///    of an [`IpAddr`] address as expected by [`FromStr`] implementation or a host
-///    name.
+///    name. The second [`u16`] value of the tuple represents a port.
```

Comment:
```suggestion
///    name. [`u16`] value of the tuple represents a port.
```
Since we don't mention "the first" it will be better off without "the second".